spec/dmv_data_service_spec.rb

```diff
@@ -42,4 +45,4 @@
       expect(@dds.mo_dmv_office_locations.size).to be_an(Integer)
```

Comment:
Further validate this data - what is the shape of the data of the first element of this collection? Test it!